### DISCUSSION TITLE
Suggested support for QR codes

### DISCUSSION THREAD

**Discussion by digger-yu** [community/community#53743] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Suggested support for QR codes

QR code is very convenient, their own account is bound to the local bank card, payment only need to scan each other's QR code can be, github's current sponsorship function is still more traditional and conservative (cumbersome) way to achieve

github has a lot of friendly users, I think if github support QR code payment, may exceed paypal and visa and other cross-border payment systems

**Comment by anjuan** [comment]
Hi @digger-yu. Thanks for the suggestion. Can you provide an example of how you've used this before? That would be super helpful for understanding how QR code payments have worked for you in the past.

**Reply by digger-yu** [reply]
In China we have a popular mobile chat tool, wechat, which has qr-code scanning and payment collection function, in fact qr-code is similar to a personal wallet address,  other people pay when you do not need to enter this wallet address, but scan the qr-code, enter the amount to complete the payment, this is a very convenient function to open and use,
The wallet can only be bound to one's bank card, and the bank card is real-name authentication, which indirectly completes the real-name authentication of wechat, similar to KYC 

I believe this is a very convenient feature, github has a lot of very friendly users, suppose I see an interesting project after, want to invite him to drink a cup of coffee, Of course, it can be any other amount, or blockchain currency (if supported) 
if there is qr-code then it will be very convenient

The traditional swift payment system is very expensive, slow, and cumbersome, you need to enter your bank card number, address, name, etc., and you may even need to enter the other party's bank code
For github, there may also be different country policies and legal risks to consider
I don't know if I explained it clearly

**Comment by anjuan** [comment, marked as answer]
Thank you for the explanation. I'll share it with the team!

**Reply by digger-yu** [reply]
Add notes
Most github users are located in different countries and regions of the world, and they may have different currencies in different countries and regions.
It is possible to set up a bank card that must be tied to the wallet after it is created.
However, most users do not necessarily withdraw their donations after receiving them.
Donations that are not withdrawn may form a pool of money

github can also be set up for some charity projects, such as UNICEF.

Assuming the wallet is a multi-currency account, users can accept donations from people in different countries
 can set the priority of using the money in the wallet when making payments
 can set it so that when a user withdraws money from the wallet to a local bank card in a different country, only a percentage of the transfer fee is charged

I see sponsors as more of an incentive for contributors rather than a major source of revenue
Smaller sponsors need to be easier to use

Looking forward to early realization ^_^
